namerd/docs/storage.md

```diff
@@ -23,19 +23,164 @@ Stores the dtab in memory.  Not suitable for production use.
 
 *experimental*
 
-Stores the dtab with the Kubernetes master via the ThirdPartyResource APIs. Requires a cluster
-running Kubernetes 1.2+ with the ThirdPartyResource feature enabled.
-
-* *host* -- Optional. The location of the Kubernetes API. (default: "kubernetes.default.svc.cluster.local")
-* *port* -- Optional. The port used to connect to the Kubernetes API. (default: 443)
-* *tls*  -- Optional. Whether to connect to the Kubernetes API using TLS. (default: true)
-* *tlsWithoutValidation* -- Optional. Whether to disable certificate checking against the Kubernetes
+Stores the dtab with the Kubernetes master via the ThirdPartyResource APIs. 
+
+**Prerequisites**  
+1. Requires a cluster running Kubernetes 1.2+.  
+2. `kubectl` client 1.3+ is installed.  
+3. [ThirdPartyResource](http://kubernetes.io/docs/api-reference/extensions/v1beta1/definitions/#_v1beta1_thirdpartyresource) feature should be enabled in Kubernetes cluster.  
+  1. **How to check ThirdPartyResource is enabled**
```

Comment:
Replaced 1. to '*'. I think this "How to check..." should be as a sub-part of item `3. [ThirdPartyResource](http://kubernetes.io/docs/api-reference/extensions/v1beta1/definitions/#_v1beta1_thirdpartyresource) feature should be enabled in Kubernetes cluster.` and a list is the one possible way to visualize it in such way.

What do you think @esbie ?